documentation/wiki/ChangeWaves-Dev.md

```diff
@@ -12,6 +12,9 @@ The opt-out comes in the form of setting the environment variable `MSBuildDisabl
 ## Choosing a Change Wave for a New Feature
 This is determined on a case by case basis and should be discussed with the MSBuild team. A safe bet would be to check our [currently active Change Waves](ChangeWaves.md#change-waves-&-associated-features) and pick the version after the latest MSBuild version. This version corresponds to the latest version of Visual Studio.
 
+### Change Wave Versioning
+Change Wave features should match the LTS version of VS they were released with. Any feature requiring a changewave during a non-LTS release of VS should use the **NEXT** version number.
+
 # Developing With Change Waves in Mind
 For the purpose of providing an example, the rest of this document assumes we're developing a feature for MSBuild version **17.4**.
```

Comment:
```suggestion
For the purpose of providing an example, the rest of this document assumes we're developing a feature for MSBuild version **17.3**.
```

?

Reply:
It's fairly obvious how 17.4 should work, whereas 17.3 requires knowing whether it's attached to 17.2 or 17.4.

Reply:
Fair, taking the commit.

Reply:
Yeah, I was wanting to exercise the "round up" path in the example.